release/release-debian

```diff
@@ -99,11 +99,11 @@ EOF
         reprepro -b $DEBIAN_REPO includedsc unstable cockpit_$debversion.dsc
 
         # build binary package
-        sudo pbuilder clean
-        sudo pbuilder update --configfile pbuilderrc --distribution unstable
-        sudo pbuilder build --configfile pbuilderrc \
-                            --hookdir /etc/pbuilder/hooks \
-                            cockpit_$debversion.dsc
+        basetgz=/var/cache/pbuilder/unstable.tgz
```

Comment:
If we want to release multiple versions in the future, then we might have two different `basetgz` here - what about calling this one `basetgz_unstable` to match the dist settings in the following lines?